spec/lib/krikri/entity_consumer_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'spec_helper'
+
+describe Krikri::EntityConsumer do
+
+  before(:all) do
+    DatabaseCleaner.clean_with(:truncation)
+    create(:krikri_harvest_activity)
+    create(:krikri_mapping_activity)
+  end
+
+  context 'as a mapper agent' do
+    let(:generator_uri) do
+      (RDF::URI(Krikri::Settings['marmotta']['ldp']) /
```

Comment:
Same comment as for [`aggregation_entity_behavior_spec.rb`](https://github.com/dpla/KriKri/pull/131/files#r27392504), without @no-reply's feedback.